tests/handlers/aws/test_handler.py

```diff
@@ -462,20 +464,6 @@ def test_lambda_handler_noop(self) -> None:
             del lambda_event["Records"][0]["messageAttributes"]["originalEventSourceARN"]
             assert handler(lambda_event, ctx) == "completed"  # type:ignore
 
-        with self.subTest("no input defined for kinesis-data-stream"):
```

Comment:
Removing this tests because it does not make sense. [Reference](https://docs.aws.amazon.com/streams/latest/dev/key-concepts.html).

We have now tests in `test_integrations` to check the input types.